manifests/config.pp

```diff
@@ -18,6 +18,10 @@
     mode  => '0600',
     owner => 'tomcat',
     group => 'tomcat',
+  } ~> exec { "restorecon -vF ${candlepin::candlepin_conf_file}":
```

Comment:
I really want to avoid adding these execs. How about we install `candlepin-selinux` in the spec helper instead?

Reply:
@ekohl will it always be the case in real world use that the `${name}-selinux` RPM is already installed? If not, I'd really prefer to make the modules actually idempotent rather than hiding the issue in the test env. But if we assume that the package is always installed already, then I'm fine with matching the same conditions in the test env.

Reply:
In the installer [have a workaround](https://github.com/theforeman/foreman-installer/blob/develop/hooks/pre/31_install_scl_packages.rb). Currently `candlepin-selinux` is not there, but the package `katello` depends on it. I think it's safe to assume it's present.

A general rule of thumb: if you use `exec` in Puppet, you're working around it and you should question if you're doing it right. In this case we set up the environment so that we don't hit the bug so I'd like to avoid adding code.